shared/src/shared.rs

```diff
@@ -43,14 +43,14 @@ pub struct FreezerClose {
 impl Drop for FreezerClose {
     fn drop(&mut self) {
         self.stopped.store(true, Ordering::SeqCst);
-        self.stop.try_send();
+        self.stop.try_send(());
     }
 }
 
 impl Drop for Shared {
     fn drop(&mut self) {
         if let Some(ref mut stop) = self.async_stop {
-            stop.try_send();
+            stop.try_send(());
```

Comment:
The unit tests hang here.

Because the tokio runtime thread couldn't `join`:
https://github.com/nervosnetwork/ckb/blob/8863dbc71708fbf2aadbf2e44be638d752d0877d/util/stop-handler/src/lib.rs#L90-L94.

But the `ChunkProcss` is waiting for this signal:
https://github.com/nervosnetwork/ckb/blob/8863dbc71708fbf2aadbf2e44be638d752d0877d/tx-pool/src/service.rs#L111-L116

This signal will be sent when `drop` the `TxPoolController`, which is after `drop(Shared)`.

So the unit tests hang: `ChunkProcss` and `Shared` both are waiting each other.

I tried call `self.chunk_stop.try_send(Command::Stop); ` before `stop.try_send(..)` in `Shared::drop`.
All unit tests are passed.